### ISSUE TITLE
wrong number of arguments (given 4, expected 3) when calling zrangebyscore

### ISSUE THREAD

**Issue by arbesulo** [nateware/redis-objects#260]
Hi, I'm getting the following error after upgrading to ruby 3.0.1

I see that there is an [open PR](https://github.com/nateware/redis-objects/pull/258), but unfortunately with no activity since the code review.

Is there any chance to address this issue by the maintainers?

```
bundle exec rake environment resque:scheduler
resque-scheduler: [INFO] 2021-04-15T07:13:03+00:00: Starting
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler/lock/base.rb:50: warning: Socket.gethostbyname is deprecated; use Addrinfo.getaddrinfo instead.
resque-scheduler: [INFO] 2021-04-15T07:13:03+00:00: Master scheduler
resque-scheduler: [INFO] 2021-04-15T07:13:03+00:00: Reloading Schedule
resque-scheduler: [INFO] 2021-04-15T07:13:03+00:00: Loading Schedule
resque-scheduler: [INFO] 2021-04-15T07:13:03+00:00: Schedule empty! Set Resque.schedule
resque-scheduler: [INFO] 2021-04-15T07:13:03+00:00: Schedules Loaded
rake aborted!
ArgumentError: wrong number of arguments (given 4, expected 3)
/usr/local/bundle/gems/redis-4.2.5/lib/redis.rb:1961:in `zrangebyscore'
/usr/local/bundle/gems/redis-objects-1.5.0/lib/redis/objects/connection_pool_proxy.rb:10:in `block in method_missing'
/usr/local/bundle/gems/connection_pool-2.2.5/lib/connection_pool.rb:63:in `block (2 levels) in with'
/usr/local/bundle/gems/connection_pool-2.2.5/lib/connection_pool.rb:62:in `handle_interrupt'
/usr/local/bundle/gems/connection_pool-2.2.5/lib/connection_pool.rb:62:in `block in with'
/usr/local/bundle/gems/connection_pool-2.2.5/lib/connection_pool.rb:59:in `handle_interrupt'
/usr/local/bundle/gems/connection_pool-2.2.5/lib/connection_pool.rb:59:in `with'
/usr/local/bundle/gems/redis-objects-1.5.0/lib/redis/objects/connection_pool_proxy.rb:10:in `method_missing'
/usr/local/bundle/gems/redis-namespace-1.8.1/lib/redis/namespace.rb:476:in `call_with_namespace'
/usr/local/bundle/gems/redis-namespace-1.8.1/lib/redis/namespace.rb:352:in `block (2 levels) in <class:Namespace>'
/usr/local/bundle/gems/resque-2.0.0/lib/resque/data_store.rb:65:in `method_missing'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler/delaying_extensions.rb:310:in `search_first_delayed_timestamp_in_range'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler/delaying_extensions.rb:114:in `next_delayed_timestamp'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler.rb:185:in `handle_delayed_items'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler.rb:68:in `block in run'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler.rb:56:in `loop'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler.rb:56:in `run'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler/cli.rb:117:in `run_forever'
/usr/local/bundle/gems/resque-scheduler-4.4.0/lib/resque/scheduler/tasks.rb:18:in `block (2 levels) in <main>'
/usr/local/bundle/bin/bundle:23:in `load'
/usr/local/bundle/bin/bundle:23:in `<main>'
Tasks: TOP => resque:scheduler
(See full trace by running task with --trace)
```

**Comment by nateware** [comment]
I just commented on the PR - looks like the fix isn't correct - would appreciate any help on this.

**Comment by nateware** [comment]
This was fixed in #258
